src/Poster.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./css/poster.css";
+
+
+const Poster = ({imagePath, title, id}) => {
```

Comment:
Thanks for catching that we did need the ID we can figure out where to assign it in the revision. Thank you! Looks great.